cmd/riscli/grpc.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"crypto/tls"
+	"os"
+
+	"github.com/bio-routing/bio-rd/util/log"
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+func SetupGrpcClient(c *cli.Context) *grpc.ClientConn {
```

Comment:
Please rename to SetupGRPCClient (see https://github.com/golang/go/wiki/CodeReviewComments#initialisms)